lang/lang_en_fr.txt

```diff
@@ -261,6 +305,7 @@
 #MSG_FSENSOR_AUTOLOAD
 "F. autoload"
 "Autochargeur"
+
```

Comment:
"F. autoload" = "F. autocharg."

Reply:
> F. autocharg.

Use this.
Also update it to c=13 after looking into the source code